idea/src/org/jetbrains/kotlin/idea/intentions/conventionNameCalls/ReplaceContainsIntention.kt

```diff
@@ -48,7 +48,8 @@ class ReplaceContainsIntention : SelfTargetingRangeIntention<KtDotQualifiedExpre
         if (!element.isReceiverExpressionWithValue()) return null
 
         val functionDescriptor = getFunctionDescriptor(element) ?: return null
-        if (!functionDescriptor.isOperator || !OperatorChecks.check(functionDescriptor).isSuccess) return null
+
+        if (!functionDescriptor.isOperator) return null
```

Comment:
I fixed it. But I have a little problem that test failed.
https://github.com/takahirom/kotlin/blob/145ee2c92f909706272922e2f3eab03d64d853ec/idea/testData/intentions/conventionNameCalls/replaceContains/withoutOperatorModifier.kt

Result:

```
java.lang.RuntimeException: java.lang.AssertionError: isAvailable() for class org.jetbrains.kotlin.idea.intentions.conventionNameCalls.ReplaceContainsIntention should return false
```

---

So I delete `IS_APPLICABLE: false` comment. And I created `withoutOperatorModifier.kt.after`.

Result:

```
java.lang.RuntimeException: org.junit.ComparisonFailure: All actual errors should be mentioned in test data with // ERROR: directive. But no unnecessary errors should be me mentioned 
Expected :<empty>
Actual   :'operator' modifier is required on 'contains' in 'test.Test'
```

---

So I added  `// ERROR: 'operator' modifier is required on 'contains' in 'test.Test'` to `withoutOperatorModifier.kt`

Result:

```
java.lang.RuntimeException: org.junit.ComparisonFailure: All actual errors should be mentioned in test data with // ERROR: directive. But no unnecessary errors should be me mentioned 
Expected :'operator' modifier is required on 'contains' in 'test.Test'
Actual   :<empty>
```

---

Probably Because of  `'operator' modifier is required on 'contains' in 'test.Test'` Error is  transformed syntax error by intention.

Do you have any idea to solve this?